README.md

```diff
@@ -91,6 +90,9 @@ pub fn main() {
 }
 ```
 
+## App/APK configuration
+Android APKs contain a file called `AndroidManifest.xml`, which has things like permission requests and feature declarations, plus configuration of intents, resource folders and more. This file is autogenerated by `cargo-apk`. To control what goes in it through Cargo.toml, refer to ['cargo-apk README.md'](./cargo-apk/README.md).
```

Comment:
```suggestion
Android APKs contain a file called `AndroidManifest.xml`, which has things like permission requests and feature declarations, plus configuration of intents, resource folders and more. This file is autogenerated by `cargo-apk`. To control what goes in it through Cargo.toml, refer to [`cargo-apk`'s README](./cargo-apk/README.md).
```

Reply:
@hrydgard If this is a change you agree with, kindly accept it so that I can merge the PR :)

Reply:
done!